library/Fission/Storage/IPFS.hs

```diff
@@ -4,6 +4,7 @@ module Fission.Storage.IPFS
   , get
   , pin
   , unpin
+  , dagput
```

Comment:
This is fine, but it can't hurt to move this to `Fission.IPFS.DAG.put`. We'll likely accrue more DAG functions as time goes on, and we lose nothing by being super clear with naming.